openlibrary-wikidata-bot/jobs/sync_author_identifiers_from_wikidata.py

```diff
@@ -0,0 +1,210 @@
+from datetime import datetime
+from olclient import OpenLibrary
+from os import makedirs
+import argparse
+import csv
+import datetime
+import json
+import logging
+import re
+import sys
+
+# Setup logger
+logger = logging.getLogger("jobs.sync_author_wikidata_ids")
+logger.setLevel(logging.DEBUG)
+# The logging format
+log_formatter = logging.Formatter('%(name)s;%(levelname)-8s;%(asctime)s %(message)s')
+# Log warnings+ to console
+console_handler = logging.StreamHandler()
+console_handler.setLevel(logging.WARN)
+console_handler.setFormatter(log_formatter)
+logger.addHandler(console_handler)
+# Log INFO+ to the log file
+log_dir = 'logs/jobs/sync_author_wikidata_ids'
+problem_dir = 'exports/jobs/sync_author_wikidata_ids'
+makedirs(log_dir, exist_ok=True)
+makedirs(problem_dir, exist_ok=True)
+log_file_datetime = datetime.datetime.now().strftime("%Y%m%d_%H%M%S")
+log_file = log_dir + '/sync_author_wikidata_ids_%s.log' % log_file_datetime
```

Comment:
Nice thinking to save the errors in a csv file! Let's put them in the logs directory as well, that way it'll get saved in the jenkins artifact
```suggestion
log_dir = 'logs/jobs/sync_author_wikidata_ids'
makedirs(log_dir, exist_ok=True)
log_file_datetime = datetime.datetime.now().strftime("%Y%m%d_%H%M%S")
log_file = log_dir + '/sync_author_wikidata_ids_%s.log' % log_file_datetime
problems_file = log_dir + '/sync_author_wikidata_ids_%s_problems.csv' % log_file_datetime
```